analysis/datetime/javatime/javatime.go

```diff
@@ -0,0 +1,243 @@
+//  Copyright (c) 2023 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package javatime
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/blevesearch/bleve/v2/analysis"
+	"github.com/blevesearch/bleve/v2/registry"
+)
+
+const Name = "javastyle"
```

Comment:
We're not going to call this "javastyle". Drop all references to "java" from function names and file names.

You can indicate in comments that you got it from oracle but choose a better name for this.

Reply:
made it iso style since its somewhat similar to iso 8601 spec. Same format is used in swift, ruby etc